runtime/java.go

```diff
@@ -69,11 +91,12 @@ func (conf javaRuntime) GetYamlTemplate() string {
 # Example value: main.Hello
 main: <name>
 
-# REQUIRED
+# OPTIONAL
 # A list of paths where classes and other resources can be found.
+# By default, the unikernel root "/" is added to the classpath.
```

Comment:
I see, but still wonder why. Is it because `java.so` is executed from `/` and is therefore the current directory used as the default classpath?

Reply:
Just to be sure: you do know that "classpath" specifies where Java can find the classes/jars to run, yes? I has to do with application structure, not with Java installation at all. And I think it's a valid guess to expect application files in the unikernel root by default, yes? I'm looking forward to be clarifying this, because I'm not sure I understand what bothers you here?😄

Reply:
As explained about, just the fact that classpath is set to `/` was bothering me. It's a long time since I was running Java so I will take your word on it 😈.